### ISSUE TITLE
Word kerning issues on Chromium browsers on Windows 10

### ISSUE THREAD

**Issue by chimera238** [knighty/fimfiction-issues#395]
I've noticed that sometimes kerning is off on certain paragraphs on stories. Occasionally, words will have huge spaces between them instead of the normal kerning. When the formatting is adjusted to change the size of the paragraph (text size, page width, display font), the kerning goes back to normal. So far, this has only occurred on Chromium-based browsers (Google Chrome, Microsoft Edge, possibly Opera but not yet tested) running Windows 10. Another user has speculated that this is related to importing from Google Docs, but when I manually typed in an improperly-kerned paragraph into a sample chapter, I still had the glitch. It might be related to either italics specifically or BBCode markup in general; I noticed that italics often appeared at the end of glitched lines and removing the italic tag usually solved it.

Here are the settings under which I'm experiencing the glitch:
- Browser: Google Chrome. Another browser like Firefox does not have the glitch.
- Operating system: Windows 10. Using Chrome on Windows 7 does not have the glitch.
- Font: Open Sans
- Text size: 1
- Color: default
- Spacing: 1.8
- Page width: 1.2
- Paragraphs: Double spaced
- Position: Justified

Under those circumstances, [this chapter](https://www.fimfiction.net/story/428991/22/how-the-tantabus-parses-sleep/quality-assurance) has several instances of this happening. Here's a specific paragraph, BBCode markup included:
- “And basilisks are supposed to be able to [i]kill[/i] you with their gaze, but the dreamer was only turned to stone and not until she started speaking.”

With this specific paragraph, I noticed that the glitch still appeared when text size was set to 1.3 or when page width was set to 1.1. Color, paragraph spacing, and text position had no effect on the glitch.

**Comment by ocalhoun6** [comment]
Yeah. This has nothing to do with importing from google docs. I never use that feature, but I've found my stories also getting messed up on Chromium browsers -- usually after an italics tag. But not _all_ italics tags. It's weird.

Removing the italics tag fixes it, but then if you put the italics tag back in any way, the problem returns.
